HST/queue_manager/task_queue_db.py

```
@@ -229,29 +271,57 @@ def erase_all_task_queue():
+    """
+    return task.execution_time is None or now >= task.execution_time
+
 def get_next_task_to_be_run():
     """
     Get the next task to be run from database. Return the table row entry.
+
+    Waiting tasks are considered in descending priority and order. Tasks with a future
+    execution_time are skipped until that time has passed.
     """
     if not db_exists():
         return
```

Ruff (0.15.17)

[warning] Do not implicitly `return None` in function able to return non-`None` value

Add explicit `None` return value

(RET502)